Description: change config file, in user dir. That make respect the freedesktop
 guidelines
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2019-11-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/client/src/game/gamesettings.cpp
+++ b/client/src/game/gamesettings.cpp
@@ -2,7 +2,7 @@

 #include <cstdint>

-#define SETTINGS_FILE "data/game.ini"
+#define SETTINGS_FILE "/dynablasterrevenge/client/game.ini"

 // defaults

@@ -119,7 +119,7 @@

 GameSettings::SettingsPrivate::SettingsPrivate()
  : QSettings(
-      SETTINGS_FILE,
+      QStandardPaths::writableLocation(QStandardPaths::ConfigLocation) + SETTINGS_FILE,
       QSettings::IniFormat
    )
 {
@@ -1685,4 +1685,3 @@
    mCameraShakeIntensity = value;
 }

-
--- a/client/src/game/gamesettings.h
+++ b/client/src/game/gamesettings.h
@@ -5,6 +5,7 @@
 #include <QColor>
 #include <QSettings>
 #include <QObject>
+#include <QStandardPaths>

 // shared
 #include "constants.h"
